File: app/src/main/java/org/blitzortung/android/jsonrpc/JsonRpcException.java
package org.blitzortung.android.jsonrpc;


public class JsonRpcException extends RuntimeException {

    private static final long serialVersionUID = -8108432807261988215L;

    public JsonRpcException(String msg, Exception e) {
        super(msg, e);
    }

    public JsonRpcException(String msg) {
        super(msg);
    }
}

File: app/src/main/java/org/blitzortung/android/jsonrpc/JsonRpcException.kt
package org.blitzortung.android.jsonrpc


class JsonRpcException : RuntimeException {

    constructor(msg: String, e: Exception) : super(msg, e) {
    }

    constructor(msg: String) : super(msg) {
    }

    companion object {
        private val serialVersionUID = -8108432807261988215L
    }
}
